docs/supported-services/beanstalk.rst

```diff
@@ -134,6 +134,12 @@ The Routing element is defined by the following schema:
     routing:
       type: <http|https>
       https_certificate # Required if you select https as the routing type
+      dns_names:
+       - <string> # Optional
+
+DNS Names
```

Comment:
Yeah, that makes sense. I think that header is an artifact from when the DNS config was more complex.

I can make that change later today.